var/spack/repos/builtin/packages/neuron/package.py

```diff
@@ -446,12 +446,24 @@ def filter_compilers(self):
             filter_file(env["CC"], cc_compiler, nrniv_makefile, **kwargs)
             filter_file(env["CXX"], cxx_compiler, nrniv_makefile, **kwargs)
 
+    @when("+python")    
+    def set_python_path(self, env):    
+        for pydir in (    
+                self.spec.prefix.lib64.python,    
+                self.spec.prefix.lib.python,    
+        ):    
```

Comment:
Embrace the sadface!

Reply:
Probably could be outdented by four spaces… fine though if flake8 passes

Reply:
You're right, missed that and a couple of trailing whitespaces coming from copy paste. It should be fine now 🙏

Reply:
I think Matthias will be happy now  with https://github.com/BlueBrain/spack/pull/644/commits/0e2f82ffc09693013b479ecb3511795dabd30131